app/validators/multiple_mfa_options_validator.rb

```diff
@@ -1,8 +0,0 @@
-# frozen_string_literal: true
-
-class MultipleMfaOptionsValidator < ActiveModel::EachValidator
```

Comment:
I think this became unused in #9788 when `PhoneDeletionForm` was removed

Reply:
Great minds (you and Vraj) think alike! 😜